docs/02-app/01-building-your-application/05-styling/01-css-modules.mdx

```diff
@@ -347,7 +347,10 @@ To maintain a predictable order, we recommend the following:
 - Extract shared styles into a separate shared component.
 - If using [Tailwind](/docs/app/building-your-application/styling/tailwind-css), import the stylesheet at the top of the file, preferably in the [Root Layout](/docs/app/building-your-application/routing/layouts-and-templates#root-layout-required).
 
-> **Good to know:** CSS ordering behaves differently in development mode, always ensure to check preview deployments to verify the final CSS order in your production build.
+> **Good to know:**
+>
+> - CSS ordering behaves differently in development mode, always ensure to check preview deployments to verify the final CSS order in your production build.
+> - You can use the [`cssChunking`](/docs/app/api-reference/next-config-js/cssChunking) option in `next.config.ts` to control how CSS is chunked.
```

Comment:
I'm not sure we want to recommend `next.config.ts` just yet since it is technically experimental, unless this is fine for now (we'll probably make it not experimental very soon).

Reply:
`.ts` will be part of the RC docs too, this mostly convention. let's stick with .js for filenames.